generative-ai-rag-llama-index/2-create-compute-instance/create-compute-instance.md

```diff
@@ -15,37 +25,47 @@
     Download SSH Private Key and Save Public Key 
     Click 'Create' <br>
 
-&nbsp;&nbsp;&nbsp;<img src="images/lab23.png" width="450" height="320"><br>
+&nbsp;&nbsp;&nbsp;<img src="images/On-Creation-VM.png" width="450" height="320"><br>
+
```

Comment:
Can you change these image tags to markdown too? 

```[required_description](images/On-Creation-VM.png)```

Above would be the syntax. Every image needs required description as well. Refer to lab 3.